meerk40t/ruida/device.py

```diff
@@ -610,7 +611,7 @@ def process(self, array):
 
             self.x = self.abscoord(array[1:6])
             self.y = self.abscoord(array[6:11])
-            self.plotcut.plot_append(self.x / UNITS_PER_uM, self.y / UNITS_PER_uM, 0)
+            self.plotcut.plot_append(int(self.x / UNITS_PER_uM), int(self.y / UNITS_PER_uM), 0)
```

Comment:
This code is wrong. Since it feeds that data straight into the spooler it expects it to be in device relevant units and if in actual units it needs to still multiply by the relevant device matrix. I'll probably need to do some work to fix this still. This part of the review is a note to remember to do that.